docs/cloud/satellites/byoc/aws.md

```diff
@@ -0,0 +1,116 @@
+# Installing BYOC in AWS
+
+This page documents the requirements and steps required to install BYOC Satellites in AWS.
+
+## Requirements
+Before you begin to provision your BYOC configuration within AWS, there are a few requirements you need to make sure you meet first:
+
+### Networking
+Because every configuration is different, using BYOC within your organization will require you to configure your networking to match your use case. You'll need to ensure that:
+
+* Traffic from Earthly clients (build machines or otherwise) can reach the satellites directly. When installing the AWS CloudFormation template, a security group rule is created that should allow the correct ingress from the specified CIDR block.
```

Comment:
```suggestion
* Traffic from Earthly clients, including CI build runners and developer workstations, can reach the satellites directly. When installing the AWS CloudFormation template, a security group rule is created that should allow the correct ingress from the specified CIDR block.
```

Reply:
Should we link to AWS docs for CloudFormation, security groups, etc, in case the user is not familiar with AWS?